### DISCUSSION TITLE
Boostrap Carousel V5 with Edge 41.16299.1004.0

### DISCUSSION THREAD

**Discussion by stephane-scj** [twbs/bootstrap#35671] [Q&A]
Hello all,
I will want to implement Bootstrap carousel V5.
When I go to the page https://getbootstrap.com/docs/5.0/components/carousel/, it works well with Chrome and Edge 97.0.1072.55, but it doesn't work with Edge 41.16299.1004.0 (the carousel does not scroll and clicking on the previous and next buttons does nothing).
Do you know why ? I read that Bootstrap V5 is compatible with Edge.

Thanks

**Comment by florianlacreuse** [comment]
The latest versions of Edge are supported but not the older ones. I'm afraid that Edge 41 is a little too old, sorry.

```
[flacreuse@localhost bootstrap-5.1.3]$ npx autoprefixer --info
Browsers:
  Chrome for Android: 96
  Firefox for Android: 95
  QQ Browser: 10.4
  UC for Android: 12.12
  Android: 96
  Baidu Browser: 7.12
  Chrome: 97, 96, 95, 94, 93, 92, 91, 90, 89, 88, 87, 86, 85, 84, 83, 81, 80, 79, 78, 77, 76, 75, 74, 73, 72, 71, 70, 69, 68, 67, 66, 65, 64, 63, 62, 61, 60
  Edge: 97, 96
  Firefox: 95, 94, 93, 92, 91, 90, 89, 88, 87, 86, 85, 84, 83, 82, 81, 80, 79, 78, 77, 76, 75, 74, 73, 72, 71, 70, 69, 68, 67, 66, 65, 64, 63, 62, 61, 60
  iOS Safari: 15.2, 15.0-15.1, 14.5-14.8, 14.0-14.4, 13.4-13.7, 13.3, 13.2, 13.0-13.1, 12.2-12.5, 12.0-12.1
  KaiOS Browser: 2.5
  Opera Mini: all
  Opera Mobile: 64
  Opera: 82, 81
  Safari: 15.2, 15.1, 15, 14.1, 14, 13.1, 13, 12.1, 12
  Samsung Internet: 15.0, 14.0

These browsers account for 93.58% of all users globally
```

https://getbootstrap.com/docs/5.1/getting-started/browsers-devices/#supported-browsers
